README.md

```diff
@@ -38,7 +38,7 @@ Options:
   --graphql-host <String>                                   (Default: 0.0.0.0)
   --graphql-port <Nullable`1>                               (Default: )
   --libplanet-node
-  --mpt                                                    Flag to turn on the Merkle trie feature. It is experimental.
+  --no-mpt                                                 Flag to turn off the Merkle trie feature.
```

Comment:
It seems updated to *Merkle Patricia Trie* from *Merkle Trie*, in other lines. Could you update it also this line?

> -  `--no-mpt`: Use legacy block state store instead of Merkle Patricia Trie for state saving.